<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>100</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btn_rem">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(2, 2, 2);</string>
         </property>
         <property name="text">
          <string>%</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btn_ce">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_c">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>C</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btn_del">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>del</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_frac">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>1/x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_sq">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>x²</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="btn_root">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>√x</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btn_div">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>/</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="btn_7">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="btn_8">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btn_9">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>9</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btn_multi">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="btn_4">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="btn_5">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="btn_6">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btn_minus">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="btn_1">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="btn_2">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btn_3">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="btn_plus">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="btn_plusMinus">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>+/-</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="btn_0">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="btn_dot">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>.</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="btn_ans">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>60</height>
          </size>
         </property>
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
